src/protocols/bitcoin/helpers/index.ts

```diff
@@ -19,18 +21,30 @@ export function isBtcAddressValid(address: string, networkType: string) {
 export function normalizeTransactionStructure(
   transaction: any,
   transactionOwner?: string,
-): any {
+): ITransaction {
+  const {
+    fee,
+    status,
+    txid,
+    vin,
+    vout,
+  } = transaction;
+
   return {
-    transactionOwner,
-    hash: transaction.txid, // TODO: we can go with additional field
-    microTime: transaction.status.block_time,
-    pending: !transaction.status.confirmed,
+    protocol: PROTOCOL_BITCOIN,
+    transactionOwner: transactionOwner as any,
```

Comment:
Is it a aeternity address @konradodo ?

Reply:
Konrad - yes, there's type mismatch for the `transactionOwner`. I could change the interface type to string, but this would affect whole app - a lot of changes would need to be made. We'll handle this after the MVP release.